mandatory/2-function-creation.js

```diff
@@ -1,12 +1,18 @@
-/*
+/*[]
 Write a function that:
 - takes an array of strings as input
 - removes any spaces in the beginning or end of the strings
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
-
+function tidyUpString(strArr) {
```

Comment:
Hi Negin, you have done a really great work, well done. I have got only feeling regarding how to shorten this code from line 10 to 13. Your code really makes sense, however why don't we write all three lines in only one line in this particular exercise in order not to repeat our code.

Reply:
Hi Askin,

Thank you for the feedback. It completely makes sense, but I am not sure how to combine the three line together. I really appreciate if you can help me with it. Thank you :)

Reply:
I think the code below could be used to combine three lines into one line.
`strArr[i] = strArr[i].trim().replace("/", "").toLowerCase();`

Reply:
Askin has made a great suggestion here. This is called `Function chaining/Method chaining`. 
https://dev.to/nedsoft/method-chaining-in-javascript-3klb

Reply:
Thank you, I've changed it